core/modules/system/system.module

```diff
@@ -837,16 +837,16 @@ function system_menu() {
     'access arguments' => array('access administration pages'),
     'file' => 'system.admin.inc',
   );
-  $items['admin/config/search/clean-urls'] = array(
-    'title' => 'Clean URLs',
-    'description' => 'Enable or disable clean URLs for your site.',
+  $items['admin/config/search/urls'] = array(
+    'title' => 'URL handling',
+    'description' => 'Handling URL-related configuration of your site.',
     'page callback' => 'backdrop_get_form',
-    'page arguments' => array('system_clean_url_settings'),
+    'page arguments' => array('system_urls_settings'),
     'access arguments' => array('administer site configuration'),
     'file' => 'system.admin.inc',
     'weight' => 5,
   );
```

Comment:
Could we add a backwards-compatible menu item here for `admin/config/search/clean-urls` that uses `backdrop_goto` as its menu callback? That will help modules that may be linking the old URL during the 1.x cycle, and we will remove it in 2.x.